src/test/java/org/fundacionjala/coding/jimmy/Kata2Test.java

```diff
@@ -0,0 +1,59 @@
+package org.fundacionjala.coding.jimmy;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Test class created by Jimmy Romero.
+ */
+public class Kata2Test {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a constructor to the class, or provide default values. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1258)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this left curly brace to the beginning of next line of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)